scripts/test.sh

```diff
@@ -3,35 +3,42 @@
 set -e
 
 : "${CARGO:=cargo}"
+: "${CARGO_COMMAND:=test}"
```

Comment:
with this it is possible to do

```
CARGO_COMMAND=build ./scripts/test.sh
```

Reply:
Super, this is nice :+1: This provides a quick way to test whether the stack builds.